test/test_config.py

```diff
@@ -9,10 +9,14 @@
 import jsonschema
 import yaml
 
+#TODO: Change to latest schema - 1-0-3? Otherwise use sets for nominal_resolution
```

Comment:
Yes I think we should update to 1-0-3

https://raw.githubusercontent.com/ACCESS-NRI/schema/4b7207e47afe402a732c58741ff66acc5f93b8cf/au.org.access-nri/model/output/experiment-metadata/1-0-3.json